packages/tokens/src/color-aliases.yaml

```diff
@@ -91,6 +91,9 @@ color:
       primary:
         value: '{ color.white.0.value }'
         dark: '{ color.black.300.value }'
+      primary-flair:
```

Comment:
Is the primary prefix here intentional? Seems in Figma these are:

```
--lp-color-ui-bg-flair
--lp-color-ui-border-flair
--lp-color-ui-fill-flair
```

Reply:
I can get rid of the `primary`, sure. I was just going off of @vroske-ld 's comments here: https://app.shortcut.com/launchdarkly/story/217595/dark-mode-tokens-missing-in-released-releases-view-column#activity-218104

Reply:
Though there's been a few other ones named `primary-flair` or `secondary-flair`, so I was just following that pattern for now.